07-Extract-NDVI-From-Rasters-in-R.Rmd

```diff
@@ -35,43 +36,45 @@ rasters in `R`.
 
 ###Goals / Objectives
 
-After completing this activity, you will know:
+After completing this activity, you will:
+
+* Be able to extract summary pixel values from a raster.
+* Save summary values to a .csv file.
+* Plot summary pixel values using `ggplot()`. 
 
-* How to extract summary pixel values from a raster.
-* 
 
 ###Things You'll Need To Complete This Lesson
 
+Please be sure you have the most current version of `R` and, preferably,
+RStudio to write your code.
+
 ###R Libraries to Install:
 
 * **raster:** `install.packages("raster")`
 * **rgdal:** `install.packages("rgdal")`
 
-
-####Tools To Install
-
-Please be sure you have the most current version of `R` and preferably
-R studio to write your code.
-
-
 ####Data to Download
+Download the NDVI raster files in this teaching dataset:
 
-Download the raster files for the Harvard Forest dataset:
-
-<a href="http://files.figshare.com/2434040/NEON_RemoteSensing.zip" class="btn btn-success"> DOWNLOAD Sample NEON Airborne Observation Platform Raster Data</a> 
+<a href=" https://ndownloader.figshare.com/files/3579867" class="btn btn-success"> Download Landsat derived NDVI raster files</a> 
 
-The LiDAR and imagery data used to create the rasters in this dataset were 
-collected over the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank" >Harvard</a> and 
-<a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >San Joaquin</a> field sites 
-and processed at <a href="http://www.neoninc.org" target="_blank" >NEON </a> 
-headquarters. The entire dataset can be accessed by request from the 
-<a href="http://www.neoninc.org/data-resources/get-data/airborne-data" target="_blank"> NEON 
-website.</a>  
+The imagery data used to create the rasters in this dataset were collected over
```

Comment:
The Landsat imagery used to create the NDVI rasters used in this lesson were collected...